heka/files/toml/output/http.toml

```diff
@@ -7,14 +7,16 @@ address = "{{ output.address }}"
 username = "{{ output.username }}"
 password = "{{ output.password }}"
 {%- endif %}
-http_timeout = {{ output.timeout }}
-method = "POST"
+http_timeout = {{ output.http_timeout|default(2000) }}
+method = "{{ output.method|default("POST") }}"
```

Comment:
There are plenty of hard coded values in templates, this patch is better than we have so far since it allows passing parameters, lets merge it ;)